404.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <title>Timelapse Feature Explorer</title>
+    <style>
+      html {
+        box-sizing: border-box;
+      }
+      html,
+      body,
+      main {
+        width: 100%;
+        height: 100%;
+        margin: 0;
+        padding: 0;
+        background-color: #ffffff;
+        color: #323233;
+        font-family: Lato, Arial, sans-serif;
+      }
+
+      #error-page-404 {
+        margin-top: 20px;
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+        align-items: center;
+
+        a {
+          color: #0099ff;
+
+          &:hover,
+          &:focus-within {
+            text-decoration: underline;
+          }
+        }
+      }
+    </style>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link href="https://fonts.googleapis.com/css2?family=Lato&display=swap" rel="stylesheet" />
+    <script type="module" src="/src/routes/gh_404.tsx"></script>
+  </head>
+  <body>
+    <div id="error-page-404">
+      <h1>Rerouting...</h1>
+      <p>Timelapse Feature Explorer encountered a 404 error when attempting to load this page.</p>
+      <p>
+        You should automatically be redirected to a working page. (If this doesn't work, your browser may have
+        JavaScript disabled, which the viewer needs to run.)
+      </p>
+      <br />
+      <p>
+        Return to our GitHub repository:
+        <a href="https://github.com/allen-cell-animated/nucmorph-colorizer" target="_self" rel="noreferrer noopener"
+          >https://github.com/allen-cell-animated/nucmorph-colorizer</a
+        >
```

Comment:
nit: odd formatting (though I see this is there in w3cv too)

Reply:
Fixed!